src/components/Dialog.js

```diff
@@ -39,30 +45,34 @@ const Dialog = ({
     transparent
     visible={visible}
   >
-    <View style={styles.container}>
-      <View style={styles.content}>
-        <View style={styles.header}>
-          <View style={[styles.headerSide, styles.headerSideLeft]}>
-            {headerLeft}
-          </View>
-          <Text type="h3" style={styles.headerTitle}>
-            {title}
-          </Text>
-          <View style={[styles.headerSide, styles.headerSideRight]}>
-            {headerRight}
+    <TouchableWithoutFeedback style={styles.backdrop} onPress={onCancel}>
+      <View style={styles.container}>
+        <TouchableWithoutFeedback>
```

Comment:
Why do we need 2 `TouchableWithoutFeedback` elements? Is this to only capture taps on the backdrop, but not on the dialog itself?